js/main.js

```diff
@@ -5210,6 +5229,47 @@ function updateSchedule() {
     updateTrackHall(track_halls, '.track-hall');
 }
 
+function eventDescription(){
+    var API_VERSION = "0.0.1";
+    var response = getScheduleAndTracks();
```

Comment:
There is no point in repeating this as it actually makes two ajax calls to fetch details from API. A better solution could be to get `tracks` inside `updateSchedule` function and make it a global variable, so that you can directly access it here.